azkaban-common/src/test/java/azkaban/server/session/SessionCacheTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2015 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.server.session;
+
+import azkaban.user.User;
+import azkaban.utils.Props;
+import azkaban.utils.PropsUtils;
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
The import organization doesn't seem to be correct.

Did you set up the style and SaveAction plugin as described in the contributor guide?

Reply:
Thanks for the feedback. I have changed the import order.

Reply:
Thanks.

Did the SaveAction plugin and style config work for you?